Containers/docker-socket-proxy/haproxy.cfg

```diff
@@ -1,5 +1,8 @@
 # Inspiration: https://github.com/Tecnativa/docker-socket-proxy/blob/master/haproxy.cfg
 
+global
+    maxconn 2000
```

Comment:
is there are reason for using this number?

Reply:
No

Reply:
do you know how many connection Appapi uses?

Reply:
@andrey18106 @bigcat88 do you know?

Reply:
Is it that necessary? AppAPI uses connection to daemon only during administration actions

Reply:
Apparently maxconn is needed on some system. See https://github.com/nextcloud/all-in-one/issues/3566

Reply:
It would be enough with reserve set it to 10 I guess.

Reply:
all right. adjusted.